packages/threat-composer/src/customTypes/threats.ts

```diff
@@ -14,23 +14,23 @@
   limitations under the License.
  ******************************************************************************************************************** */
 import { z } from 'zod';
-import { EntityBaseSchema, StatusSchema } from './entities';
+import { EntityBaseSchema, StatusSchema, MetadataSchemaFull } from './entities';
 import { SINGLE_FIELD_INPUT_MAX_LENGTH, LEVEL_HIGH, LEVEL_MEDIUM, LEVEL_LOW, LEVEL_NOT_SET, STATUS_NOT_SET, THREAT_STATUS_IDENTIFIED, THREAT_STATUS_NOT_USEFUL, THREAT_STATUS_RESOLVED } from '../configs';
 import threatStatus from '../data/status/threatStatus.json';
 
 export const ThreatStatementDisplayTokenSchema = z.object({
```

Comment:
Probably we should not publish the schema for the internal properties. A way to implement this is to provide a meta value and omit the property with the meta value when generate schema.

Reply:
I presume you are referring specifically and solely to `ThreatStatementDisplayTokenSchema`? I'll look into zod docs to see if there is a good way of adding some metadata that I can build conditional off for the JSON schema generation script